contrib/mesos/pkg/offers/offers.go

```diff
@@ -441,7 +441,7 @@ func (s *offerStorage) ageOffers() {
 }
 
 func (s *offerStorage) nextListener() *offerListener {
-	obj := s.listeners.Pop()
+	obj := s.listeners.Pop(nil)
```

Comment:
I agree with @karlkfi on this one. Quoting https://golang.org/doc/faq:  Experience with other languages told us that having a variety of methods with the same name but different signatures was occasionally useful but that it could also be confusing and fragile in practice. Matching only by name and requiring consistency in the types was a major simplifying decision in Go's type system.

Hence "matching by name" is idiomatic. `CancellablePop` and `Pop` have distinct semantic meanings while `Pop(nil)` is quite confusing.